mediffusion/default_config/default.yaml

```
@@ -19,6 +19,9 @@ validation:
   classifier_cond_scale: 4
   protocol: DDIM100
   log_original: false
+  log_concat: false
+  cls_log_indices: -1 # log the entire cls vector; alternatively, provide a list of desired indices to log
```

Could you please remove the comment in the file and add it to the table in the readme?

```suggestion
  cls_log_indices: -1
```